src/main/java/edu/harvard/iq/dataverse/Dataverse.java

```diff
@@ -109,37 +109,48 @@ public void setDataverseType(DataverseType dataverseType) {
     @Transient
     private final String uncategorizedString = "Uncategorized";
 
-        public String getFriendlyCategoryName(){
-       switch (this.dataverseType) {
-           case RESEARCHERS:
-               return BundleUtil.getStringFromBundle("dataverse.type.selectTab.researchers", null , "en");
-           case RESEARCH_PROJECTS:
-               return BundleUtil.getStringFromBundle("dataverse.type.selectTab.researchProjects",null, "en");
-           case JOURNALS:
-               return BundleUtil.getStringFromBundle("dataverse.type.selectTab.journals", null, "en");
-           case ORGANIZATIONS_INSTITUTIONS:
-               return BundleUtil.getStringFromBundle("dataverse.type.selectTab.organizationsAndInsitutions",null, "en");
-           case TEACHING_COURSES:
-               return BundleUtil.getStringFromBundle("dataverse.type.selectTab.teachingCourses",null, "en");
-           case LABORATORY:
-               return BundleUtil.getStringFromBundle("dataverse.type.selectTab.laboratory",null, "en");
-           case RESEARCH_GROUP:
-               return BundleUtil.getStringFromBundle("dataverse.type.selectTab.researchGroup",null, "en");
-           case DEPARTMENT:
-               return BundleUtil.getStringFromBundle("dataverse.type.selectTab.department",null, "en");
-           case UNCATEGORIZED:
+    public String getFriendlyCategoryName(){
+        String key = getFriendlyCategoryKey();
+
+        if (key.equals(uncategorizedString)) {
```

Comment:
@JayanthyChengan why would this not use a value from the bundle? I *think* in the old code, it may have been because this was used by the getIndexable... function and that wanted to make sure it could have a set string and know to ignore it? (this is a guess). But now the getIndexable method uses the key method, the friendly method could treat it the same (and replace with localized text), while the getindexable can do what it needs to and ignore it (based on ley value, instead if text value. Does that make sense?

(I've asked the team if they know of anywhere else where we would hide "Uncategorized" besides facets)

Reply:
@scolapasta  - replaced "uncategorizedString" string with bundle key "dataverse.type.selectTab.uncategorized"